BRANCHING-MODEL.md

```diff
@@ -3,20 +3,25 @@ This section describes
 - how branches are handled
 - defectdojo release cycle
 
+Please be careful to submit your pull requests to the correct branch: 
+- bugfix: latest  "release/x.y.z" branch
+- evolutions: dev branch
+- hotfixes: master branch
+
+If in doubt please use dev branch.
+
 # Release and hotfix model
 ![Schemas](doc/branching_model.png)
 ## Releasing
-- Start a release/x.y.0 release branch off dev branch
-- Commit only bug fixes from `dev` branch onto this branch
+- Start an x.y.0 release branch off dev branch
+- Commit only bug fixes onto this branch (and merge those fixes back to dev)
```

Comment:
this is not 100% clear. The release branch will need to be merged back into dev after the release. If bugfixes are needed in dev earlier, the whole release branch needs to be merged back into dev.

Reply:
i've removed that text so that we can concentrate on the diagram where all the information is already